src/core/dev/ring_simple.cpp

```diff
@@ -17,7 +17,7 @@
 #define MODULE_HDR MODULE_NAME "%d:%s() "
 
 #define ALIGN_WR_DOWN(_num_wr_) (std::max(32, ((_num_wr_) & ~(0xf))))
-#define RING_TX_BUFS_COMPENSATE 256U
+#define RING_TX_BUFS_COMPENSATE 1024U
```

Comment:
Should we add conf param instead?

Reply:
Not in current PR. Future PR will address buffer management and will use 256KB TX buffers, so original 256 hardcoded value should be ok there. I will remove this commit and address it in other PR.